variables.tf

```diff
@@ -29,8 +29,20 @@ variable "users" {
   description = "A map with transfer users and configuration details"
 }
 
-variable "vpc_endpoint_id" {
+variable "vpc_id" {
   type        = string
   default     = null
-  description = "An optional VPC endpoint ID"
+  description = "An optional VPC ID"
+}
+
+variable "subnet_ids" {
+  type        = list(string)
+  default     = null
+  description = "Optional Subnet IDs"
+}
+
+variable "address_allocation_ids" {
+  type        = list(string)
+  default     = null
+  description = "Optional Public Elastic IPs"
```

Comment:
Please order the variables alphabetically.

Additionally I wonder if this shouldn't be just one variable that contains an object. Can these variables be used independently? What if I set a `vpc_id` but omit the others? Will it still work in that case?

Reply:
if endpoint_type 'VPC' is used, you require a vpc_id, subnet(s) and eip(s). If endpoint_type 'PUBLIC' (the default) is used, you should not have a vpc_id, subnet(s) and eip(s). So I should change the code (if endpoint_type is VPC then enter vpc_id etc)

Reply:
Let me call you...